### PR TITLE
Feature/2686: Apply Import Validation Checks (DTO) for RECT DUCT WAF Data to Import API

### PR DESCRIPTION
**Pull request by subrataroyCVP**
### [Feature/2686: Apply Import Validation Checks (DTO) for RECT DUCT WAF Data to Import API](https://app.zenhub.com/workspaces/emissioners-scrum-board-5f36cd263511ad001a777f8e/issues/us-epa-camd/easey-ui/2686)

### CODE REVIEW

File: src/dtos/duct-waf-base.dto.ts [hunk 1]
```diff
@@ -1,48 +1,100 @@
...
 
   @ApiProperty({
     description: propertyMetadata.ductWafDTONumberOfTestRuns.description,
     example: propertyMetadata.ductWafDTONumberOfTestRuns.example,
     name: propertyMetadata.ductWafDTONumberOfTestRuns.fieldLabels.value,
   })
+  @IsInt()
+  @IsAtMostDigits(2, {
+    message: (args: ValidationArguments) => {
+      return `${args.property} [RECTDUCTWAF-FATAL-A] The value for ${args.value} in the Rectangular Duct Waf record ${args.property} must be 5 digits or less`;
```
Comment (jeffreyaggrey) [inline src/dtos/duct-waf-base.dto.ts]: change error message to 2 digits or less

File: src/dtos/duct-waf-base.dto.ts [hunk 2]
```diff
@@ -52,13 +104,25 @@ export class DuctWafBaseDTO {
     name:
       propertyMetadata.ductWafDTONumberOfTraversePointsWaf.fieldLabels.value,
   })
+  @IsInt()
+  @IsAtMostDigits(2, {
+    message: (args: ValidationArguments) => {
+      return `${args.property} [RECTDUCTWAF-FATAL-A] The value for ${args.value} in the Rectangular Duct Waf record ${args.property} must be 5 digits or less`;
```
Comment (jeffreyaggrey) [inline src/dtos/duct-waf-base.dto.ts]: change error message to 2 digits or less

File: src/dtos/duct-waf-base.dto.ts [hunk 3]
```diff
@@ -52,13 +104,25 @@ export class DuctWafBaseDTO {
...
 
   @ApiProperty({
     description: propertyMetadata.ductWafDTONumberOfTestPorts.description,
     example: propertyMetadata.ductWafDTONumberOfTestPorts.example,
     name: propertyMetadata.ductWafDTONumberOfTestPorts.fieldLabels.value,
   })
+  @IsInt()
+  @IsAtMostDigits(2, {
+    message: (args: ValidationArguments) => {
+      return `${args.property} [RECTDUCTWAF-FATAL-A] The value for ${args.value} in the Rectangular Duct Waf record ${args.property} must be 5 digits or less`;
```
Comment (jeffreyaggrey) [inline src/dtos/duct-waf-base.dto.ts]: change error message to 2 digits or less

File: src/dtos/duct-waf-base.dto.ts [hunk 4]
```diff
@@ -68,20 +132,52 @@ export class DuctWafBaseDTO {
     name:
       propertyMetadata.ductWafDTONumberOfTraversePointsRef.fieldLabels.value,
   })
+  @IsInt()
+  @IsAtMostDigits(2, {
+    message: (args: ValidationArguments) => {
+      return `${args.property} [RECTDUCTWAF-FATAL-A] The value for ${args.value} in the Rectangular Duct Waf record ${args.property} must be 5 digits or less`;
```
Comment (jeffreyaggrey) [inline src/dtos/duct-waf-base.dto.ts]: change error message to 2 digits or less

**Review by jeffreyaggrey — COMMENTED** [review]
Check comments for changes

File: src/dtos/duct-waf-base.dto.ts (hunk 1 above)
Comment (subrataroyCVP) [reply src/dtos/duct-waf-base.dto.ts]: ✔

File: src/dtos/duct-waf-base.dto.ts (hunk 2 above)
Comment (subrataroyCVP) [reply src/dtos/duct-waf-base.dto.ts]: ✔

File: src/dtos/duct-waf-base.dto.ts (hunk 3 above)
Comment (subrataroyCVP) [reply src/dtos/duct-waf-base.dto.ts]: ✔

File: src/dtos/duct-waf-base.dto.ts (hunk 4 above)
Comment (subrataroyCVP) [reply src/dtos/duct-waf-base.dto.ts]: ✔

**Review by jeffreyaggrey — APPROVED** [review]

**Pull request merged**